boards/f4vi1/include/periph_conf.h

```diff
@@ -28,25 +28,34 @@ extern "C" {
 #endif
 
 /**
- * @name    Clock system configuration
+ * @name Clock settings
+ *
+ * @note This is auto-generated from
+ *       `cpu/stm32_common/dist/clk_conf/clk_conf.c`
  * @{
  */
+/* give the target core clock (HCLK) frequency [in Hz],
+ * maximum: 168MHz */
+#define CLOCK_CORECLOCK      (168000000U)
```

Comment:
your tool breaks the indention :-) The inserted extra space is off by one.

Reply:
Should be fixed